sdk/core/System.Net.ClientModel/api/System.Net.ClientModel.net6.0.cs

```diff
@@ -0,0 +1,407 @@
+        public abstract System.Net.ClientModel.Core.MessageHeaders Headers { get; }
+        public bool IsError { get { throw null; } }
+        public abstract string ReasonPhrase { get; }
+        public abstract int Status { get; }
+        public abstract void Dispose();
+    }
+    [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("The constructors of the type being deserialized are dynamically accessed and may be trimmed.")]
+    public partial class ModelJsonConverter : System.Text.Json.Serialization.JsonConverter<System.Net.ClientModel.Core.IJsonModel<object>>
+    {
+        public ModelJsonConverter() { }
+        public ModelJsonConverter(System.Net.ClientModel.ModelReaderWriterFormat format) { }
+        public ModelJsonConverter(System.Net.ClientModel.ModelReaderWriterOptions options) { }
+        public System.Net.ClientModel.ModelReaderWriterOptions ModelReaderWriterOptions { get { throw null; } }
+        public override bool CanConvert(System.Type typeToConvert) { throw null; }
+        public override System.Net.ClientModel.Core.IJsonModel<object> Read(ref System.Text.Json.Utf8JsonReader reader, System.Type typeToConvert, System.Text.Json.JsonSerializerOptions options) { throw null; }
+        public override void Write(System.Text.Json.Utf8JsonWriter writer, System.Net.ClientModel.Core.IJsonModel<object> value, System.Text.Json.JsonSerializerOptions options) { }
+    }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Class)]
+    public sealed partial class ModelReaderProxyAttribute : System.Attribute
+    {
+        public ModelReaderProxyAttribute([System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.NonPublicConstructors | System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicConstructors)] System.Type proxyType) { }
+        [System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.NonPublicConstructors | System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicConstructors)]
+        public System.Type ProxyType { get { throw null; } }
+    }
+    public abstract partial class PipelineEnumerator
+    {
+        protected PipelineEnumerator() { }
+        public abstract int Length { get; }
+        public abstract bool ProcessNext();
+        public abstract System.Threading.Tasks.ValueTask<bool> ProcessNextAsync();
+    }
+    public partial class PipelineOptions
+    {
+        public PipelineOptions() { }
+        public System.Net.ClientModel.Core.PipelinePolicy? LoggingPolicy { get { throw null; } set { } }
+        public virtual System.Net.ClientModel.Core.MessageClassifier? MessageClassifier { get { throw null; } set { } }
+        public System.TimeSpan? NetworkTimeout { get { throw null; } set { } }
+        public System.Net.ClientModel.Core.PipelinePolicy[]? PerCallPolicies { get { throw null; } set { } }
```

Comment:
naming idea: `PerMessagePolicies` ? Seems to be more intuitive than thinking through the difference between a "call" and a "try"

Reply:
Message is a noun, so I'm not sure "PerMessage" works, but it's a good callout that we've got this word/concept "Call" - it'd be good to review where this shows up and see if it's the right word and/or we're stuck with it already because of precedent in GA'ed types.  We also say "invocation" sometimes, and RequestOptions/RequestContext let us override pipeline values for the duration of a call/invocation/pipeline.Send -- it'd be nice to unify this story somehow, but dunno where that will ultimately land per prioritization.  Feel free to keep sharing ideas here!